tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/QueueAutoCreationTest.java

```diff
@@ -133,6 +133,36 @@ public void testAutoCreateOnTopic() throws Exception {
       Assert.assertTrue(((ActiveMQConnection)connection).containsKnownDestination(addressName));
    }
 
+   @Test //(timeout = 30000)
+   // QueueAutoCreationTest was created to validate auto-creation of queues
+   // and this test was added to validate a regression: https://issues.apache.org/jira/browse/ARTEMIS-2238
+   public void testAutoCreateOnAddressOnly() throws Exception {
```

Comment:
this is an incomplete test.. I will finish this before merging

Reply:
please address my other comments before merging

Reply:
@michaelandrepearce I thought by adding your commit they would be fixed? the only missing point was a test timeout that I'm reverting.

Reply:
Indeed they are i wrote this before that.